collect_app/src/main/java/org/odk/collect/android/dao/InstancesDao.java

```diff
@@ -363,9 +363,8 @@ public ContentValues getValuesFromInstanceObject(Instance instance) {
         values.put(InstanceColumns.STATUS, instance.getStatus());
         values.put(InstanceColumns.LAST_STATUS_CHANGE_DATE, instance.getLastStatusChangeDate());
         values.put(InstanceColumns.DELETED_DATE, instance.getDeletedDate());
+        values.put(InstanceColumns.GEOMETRY, instance.getGeometry());
```

Comment:
Interesting! Does this mean that the `getValuesFromInstanceObject` method wasn't used in that context before?

Reply:
Yes it was used in some tests but didn't verify those new columns.